test/unit/test_sap_cli_srvb.py

```
@@ -0,0 +1,323 @@
+        fake_srvb.return_value.add_service.call_args.assert_called_once_with(SERVICE_BINDING_NAME, 'ZSAPCLI_TEST_SRVD', '0002')
+
+
+class TestSRVBRead(unittest.TestCase):
+
+    def test_cli_srvb_read_prints_summary(self):
+        conn = Connection([
+            Response(text=SERVICE_BINDING_ADT_GET_V4_XML, status_code=200,
+                     headers={'Content-Type':
+                              'application/vnd.sap.adt.businessservices.servicebinding.v2+xml; charset=utf-8'}),
+            Response(text=SERVICE_GROUP_ODATAV4_GET_XML, status_code=200,
+                    headers={'Content-Type':
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)